src/external/glfw/include/GLFW/glfw3.h

```diff
@@ -1,5 +1,5 @@
 /*************************************************************************
- * GLFW 3.4 - www.glfw.org
```

Comment:
This is not a downgrade, as GLFW currently uses the not yet released 3.4 version number on its master branch, which is in turn currently used by raylib.